config/locales/en.yml

```
@@ -201,3 +201,14 @@ en:
           - "Yes but I’m concerned about the safety of someone else"
           - "No"
           - "Not sure"
+      need_help_with:
+        title: "What do you need to find help with?"
+        options:
+          - "Feeling unsafe"
+          - "Paying bills"
+          - "Getting food"
+          - "Being unemployed or not having any work"
+          - "Going in to work"
+          - "Having somewhere to live"
+          - "Mental health and wellbeing"
+          - "I'm not sure"
```

The wording need tweaking, but is it worth highlighting the 'Not sure' option in the error message?

```suggestion
          - "I'm not sure"
        custom_select_error: 'Please choose at least one option. If you don't know, please choose "Not sure"'
```

---

Typographical quotes for apostrophes at all times please.